podvm/files/etc/systemd/system/attestation-agent.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
We need to enable this to be started when machine boots up.

Reply:
@surajssd the next patch does provide a symlink to start this on boot, right ? Also, will this service always run by default ? What happends if the `attestation-agent` command in the script fails ?